janusgraph-hadoop-parent/janusgraph-hadoop-core/src/main/java/org/janusgraph/hadoop/formats/cassandra/CqlBridgeRecordReader.java

```diff
@@ -0,0 +1,377 @@
+// Copyright 2017 JanusGraph Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
```

Comment:
@mbrukman The [Cassandra NOTICE.txt](https://github.com/apache/cassandra/blob/trunk/NOTICE.txt) doesn't look like it lists any individual contributors as copyright holders and only calls out additional copyrights for third party libs. Comparing that list to this class it looks like only google-collections (Guava) is relevant, and that's already listed in the JanusGraph NOTICE file (though not with the copyright information as provided in the Cassandra version). I also don't see an obvious AUTHORS/CONTRIBUTORS list in the project.